airbyte-integrations/connectors/source-mssql/src/testFixtures/java/io/airbyte/integrations/source/mssql/MsSQLTestDatabase.java

```diff
@@ -268,9 +268,20 @@ public MsSQLConfigBuilder withoutSsl() {
       return withSsl(Map.of("ssl_method", "unencrypted"));
     }
 
-    @Override
-    public MsSQLConfigBuilder withSsl(Map<Object, Object> sslMode) {
-      return with("ssl_method", sslMode);
+    public MsSQLConfigBuilder withEncrytedTrustServerCertificate() {
+      return withSsl(Map.of("ssl_method", "encrypted_trust_server_certificate"));
+    }
+
+
+    public MsSQLConfigBuilder withEncrytedVerifyServerCertificate(String certificate, String hostnameInCertificate) {
+      if (hostnameInCertificate != null) {
```

Comment:
Please try to avoid `null` values as function arguments, especially when the function is effectively part of an interface. 

Either define two methods instead (probably fine in this case), or use a better type if you really have to have one method (in this case `Optional<String>`).